core/modules/widgets/range.js

```diff
@@ -92,16 +173,30 @@ RangeWidget.prototype.handleInputEvent = function(event) {
 Compute the internal state of the widget
 */
 RangeWidget.prototype.execute = function() {
+	// TODO remove the next 2 lines once IE is gone!
+	this.mouseUp = true; // Needed for IE10
+	this.isIE = (/msie|trident/i.test(window.navigator.userAgent)) ? true : false;
+
+console.log("isIE: ", this.isIE)
+
 	// Get the parameters from the attributes
 	this.tiddlerTitle = this.getAttribute("tiddler",this.getVariable("currentTiddler"));
-	this.tiddlerField = this.getAttribute("field");
+	this.tiddlerField = this.getAttribute("field","text");
 	this.tiddlerIndex = this.getAttribute("index");
 	this.minValue = this.getAttribute("min");
 	this.maxValue = this.getAttribute("max");
 	this.increment = this.getAttribute("increment");
-	this.defaultValue = this.getAttribute("default");
+	this.defaultValue = this.getAttribute("default","");
 	this.elementClass = this.getAttribute("class","");
 	this.isDisabled = this.getAttribute("disabled","no");
+	// Actions since 5.1.23
+	// Next 3 only fire once!
+	this.actionsMouseDown = this.getAttribute("actionsMouseDown","");
+	this.actionsMouseUp = this.getAttribute("actionsMouseUp","");
+	// Change only fires, if start-value is different to end-value
+	this.actionsChange = this.getAttribute("actionsChange","");
+	// input fires very often!!
```

Comment:
It's a small thing but we do try to capitalise comments as sentence case to keep things neat